### PR TITLE
Adds the ability to partially understand other languages and a bit more

### PR DESCRIPTION
**Pull request by Alberyk**
Ports the feature added in Baystation12/Baystation12#19868, it allows people that knows a certain language to understand some words from another language, without needing to have it. It is used in all tajaran and unathi languages for now, Already consulted the lore team and it seems this is all they want so far.

Also, it adds Delvahhi, a zhan language.

### CODE REVIEW

**Comment by Fire-and-Glory** [comment]
changelog when

**Comment by ProbablyMenown** [comment]
You spelled Devahhi wrong.

**Comment by Alberyk** [comment]
This page: https://wiki.aurorastation.org/index.php?title=Tajaran_Ethnicities#Zhan-Khazan uses Delvahhi however.

File: code/__defines/species_languages.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 #define LANGUAGE_SIIK_TAJR "Siik'tajr"
 #define LANGUAGE_SIGN_TAJARA "Nal'rasan"
 #define LANGUAGE_YA_SSA "Ya'ssa"
+#define LANGUAGE_DELVAHII "Delvahhi"
```
Comment (skull132) [inline code/__defines/species_languages.dm]: How many fucking languages do the cats have seriously holy shit.

**Pull request merged**